src/Pwned.php

```diff
@@ -56,6 +56,8 @@ private function query($prefix)
         return Cache::remember('pwned:'.$prefix, Carbon::now()->addWeek(), function () use ($prefix) {
             $curl = curl_init('https://api.pwnedpasswords.com/range/'.$prefix);
             curl_setopt($curl, CURLOPT_RETURNTRANSFER, true);
+            //Set Add-Padding to true to pad queries. (See: https://haveibeenpwned.com/API/v3#PwnedPasswordsPadding)
+            curl_setopt($curl, CURLOPT_HEADER, Add-Padding: false);
```

Comment:
Great idea to add this!

Am I missing something or does this need quotes around the header value?

Reply:
I may have messed it up, as I'm not real familiar with PHP. Looking over a few examples in PHPs Manual, however, it does look like that's the case. So it probably should be:
`curl_setopt($curl, CURLOPT_HEADER, "Add-Padding: false");`

Reply:
Do you think you will update this PR @tehCh0nG or @valorin or could I make a new one, I'd love to have this.

Reply:
Oh oops, I'm so sorry @nathanheffley. I completely forgot about this. I'll fix it up now and get it working and merge in. :-)

Reply:
No need to be sorry, just wanted to help push this over the finish line if possible 🙂

Reply:
The nudge was much appreciated @nathanheffley 😁
I've merged and tagged it as `1.3.4`, so you should be able to pull it down via composer now.

Reply:
Just updated and it works great, thank you!